tools/Makefile.shared

```diff
@@ -70,62 +122,35 @@ clean::
 	rm -f ocamldep.opt
 
 
-INSTALL_BINDIR=$(DESTDIR)$(BINDIR)
-INSTALL_LIBDIR=$(DESTDIR)$(LIBDIR)
-
-install::
-	cp ocamldep "$(INSTALL_BINDIR)/ocamldep$(EXE)"
-	if test -f ocamldep.opt; then \
-	  cp ocamldep.opt "$(INSTALL_BINDIR)/ocamldep.opt$(EXE)"; else :; fi
-
 # The profiler
 
 CSLPROF=ocamlprof.cmo
 CSLPROF_IMPORTS=misc.cmo config.cmo identifiable.cmo numbers.cmo \
-  arg_helper.cmo clflags.cmo terminfo.cmo \
-  warnings.cmo location.cmo longident.cmo docstrings.cmo \
-  syntaxerr.cmo ast_helper.cmo parser.cmo lexer.cmo parse.cmo
+		arg_helper.cmo clflags.cmo terminfo.cmo	warnings.cmo \
+		location.cmo longident.cmo docstrings.cmo syntaxerr.cmo \
+		ast_helper.cmo parser.cmo lexer.cmo parse.cmo
```

Comment:
Please get rid of all the whitespace changes, they're a pain in the ass when trying to trace back changes with `git blame` and `git log`. I would much rather have inconsistent indentation and line length than a lot of small cosmetic changes in the history.